.github/ISSUE_TEMPLATE/Bug_Report.md

```diff
@@ -1,15 +1,13 @@
 ---
 name: Bug Report
 about: Create a bug report
-
 ---
 
 <!-- Have a question? Join #thelounge on freenode -->
 
-* *Node version:* 
-* *Browser version:* 
-* *Device, operating system:* 
-* *The Lounge version:* 
+- _Node version:_
```

Comment:
Hmmmmm.

Reply:
Yeah, that's definitely an opinion, but even though `*` can be used for italics (resp. `__` for bold), it's somewhat common to prefer `_` instead (resp. `**` for bold). Example, hitting <kbd>Ctrl</kbd>+<kbd>B</kbd> on a GitHub comment uses `**` and hitting <kbd>Ctrl</kbd>+<kbd>I</kbd> uses `_`.

We're using that format _almost_ everywhere anyway (website included), so that's nice to stick to a single syntax.